.rubocop.yml

```diff
@@ -1,21 +1,41 @@
+inherit_from: .rubocop_todo.yml
+
 require:
   - rubocop-rails
+  - rubocop-rspec
+  - rubocop-performance
 
 AllCops:
-  TargetRubyVersion: 2.5
-  NewCops: disable
+  TargetRubyVersion: 3.0
+  NewCops: enable
+  DisplayCopNames: true
+  DisplayStyleGuide: true
+  ExtraDetails: true
+  UseCache: true
+  CacheRootDirectory: tmp
   Exclude:
-    - 'spec/**/*'
-    - 'db/**/*'
-    - 'app/views/**/*'
-    - 'config/**/*'
-    - 'bin/*'
-    - 'Rakefile'
-    - 'node_modules/**/*'
-    - 'Vagrantfile'
-    - 'vendor/**/*'
-    - 'lib/json_ld/*'
-    - 'lib/templates/**/*'
+    - bin/*
+    - build/**/*
+    - chart/**/*
+    - config.ru
+    - config/boot.rb
+    - config/environment.rb
+    - config/puma.rb
+    - coverage/**/*
+    - db/schema.rb
+    - dist/**/*
+    - log/**/*
+    - node_modules/**/*
+    - public/**/*
+    - Rakefile
+    - redis/**/*
+    - streaming/**/*
+    - templates/**/*
+    - tmp/**/*
+    - Vagrantfile
+    - vendor/**/*
```

Comment:
This folder is actually empty. We can delete it.
- https://github.com/mastodon/mastodon/pull/20941

Reply:
I suppose it might get regenerated during a Rails upgrade via `rails app:update`.

Reply:
I thing it is comming with rails new, because it is a standard rails assets directory, that comes before gems but after app/assets in the precedence:
https://guides.rubyonrails.org/asset_pipeline.html#asset-organization